README.md

```diff
@@ -60,3 +57,14 @@ pip install -e .
 
 in your git clone and then test your changes in Spyder. We follow PEP8 and
 PEP257 style guidelines.
+
```

Comment:
I took the liberty of adding the sponsors section to the repo

Reply:
Ok, no problem :+1: